src/pages/index.js

```
@@ -73,409 +74,1029 @@ const LandingPage = () => {
+
+              {/* Description - Responsive */}
+              <p className="text-base sm:text-lg md:text-xl text-gray-600 mt-4 sm:mt-6 mb-8 sm:mb-10 max-w-3xl mx-auto px-4">
+                The first annual on-chain awards show celebrating the people and
+                projects of NEAR and predicting achievements in the upcoming
+                year
+              </p>
+
+              <div className="flex flex-col sm:flex-row gap-4 justify-center">
+                <button
+                  onClick={() => (window.location.href = "/nomination")}
+                  className="group inline-flex items-center justify-center px-8 py-4 text-lg font-medium text-white bg-gradient-to-r from-blue-500 to-teal-500 rounded-xl shadow-lg hover:shadow-xl transform hover:-translate-y-0.5 transition-all duration-200"
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)